Makefile

```diff
@@ -47,6 +47,11 @@ build_sasclient_egg: $(ENV_DIR)/bin/python setup.py
 
 .PHONY: build_sasclient_wheel
 build_sasclient_wheel: $(ENV_DIR)/bin/python setup.py
+
+	# Ensure we have an up to date version of pip with wheel support
+	${ENV_DIR}/bin/pip install --upgrade pip==9.0.1
+	${ENV_DIR}/bin/pip install wheel==0.30.0
```

Comment:
Should these be installed as part of the `env` target?  That's where we install `setuptools` (which is also used by this target).

Reply:
We could - but we only need them for building a wheel, so it seems neater to do them here.